### DISCUSSION TITLE
Mineral Resources - sourceReference specified as DocumentCitation in Data Spec but not in Schema

### DISCUSSION THREAD

**Discussion by robblis** [INSPIRE-MIF/helpdesk-validator#797] [Q&A] [answered]
Hi Inspire Team,

In the Mineral Resources theme, the sourceReference attribute for EarthResource, OreMeasure, and Mine features is defined as a DocumentCitation type in the Data Specification. However, in the MineralResourcesCore.xsd the schema only defines a reference an xlink attribute.

Is this intentional or was something overlooked in the schema?

**Comment by fabiovinci** [comment, marked as answer]
Dear @robblis,

it is a known bug in the schema and will be fixed in the next schema release at the end of July.

See the related issue: https://github.com/INSPIRE-MIF/application-schemas/issues/41
